src/onefetch/cli.rs

```diff
@@ -208,6 +209,14 @@ impl Cli {
             .long("no-merges")
             .help("Ignores merge commits."),
         )
+        .arg(
+            Arg::with_name("no-bots")
+            .long("no-bots")
+            .min_values(0)
+            .max_values(1)
+            .value_name("REGEX")
+            .help("Exclude [bot] commits. Use <REGEX> to override the default pattern."),
```

Comment:
I just realized that maybe a validator should be included to prevent invalid regex, which could cause a panic [here](https://github.com/o2sh/onefetch/pull/448/files#diff-f6a4b0b4b4baa74961fd54b0f78627290079064c0c0c90281d0a9a789f85c362R288)

Reply:
```
.validator(|p| {
    if Regex::from_str(&p).is_err() {
        return Err(String::from("must be a valid regex pattern"));
    } else {
        Ok(())
    }
}),
```